src/main/resources/resources/ManagerResources_pt.properties

```diff
@@ -1746,4 +1746,5 @@ info.manage.associated.objects.no.administrativeOffices=Não existem secretarias
 info.manage.associated.objects.no.emptyDegree=Não existe curso de unidades isoladas.
 title.manage.associated.objects.departments.edit=Editar Departamento
 title.manage.associated.objects.administrativeOffices.create=Criar Secretaria
-info.manage.associated.objects.emptyDegree.associated.with=Curso de unidades isoladas associado com:
\ No newline at end of file
+info.manage.associated.objects.emptyDegree.associated.with=Curso de unidades isoladas associado com:
+warning.manager.SIBS.outdate.code=O código de pagamento {0} ({1}) é antigo e já devia estar integrado. Analisar manualmente.
```

Comment:
@sergiofbsilva There is a typo in the key here. It should be "warning.manager.SIBS.outdated.code" 🔍

Reply:
@danielcorreia96 e vires cá este ano não ? 😆